README.md

```diff
@@ -28,7 +28,6 @@ maintained by Informal Systems.
 
 #### P2P Messages
 
-- [Peer Exchange (PEX)](./spec/p2p/messages/pex.md): gossip known peer addresses so peers can find each other
```

Comment:
This file still exists, any reason to delete this line?

Reply:
Nope, just got a little over-excited. Fix'd.